src/mainwindow.ui

```diff
@@ -117,10 +117,17 @@
     <addaction name="separator"/>
     <addaction name="action_anim_delete"/>
    </widget>
+   <widget class="QMenu" name="menuUtilities">
+    <property name="title">
+     <string>Utilities</string>
```

Comment:
Maybe I thought about it backwards. I've got another feature in the works that wouldn't fit anywhere else.
I will change the rendering so that it only displays visible Voxels (unless wireframe mode). That feature could also be used to make Objects hollow.
I'm also thinking about turning frames into layers (that could also function as frames), if rendering speed gets fast enough.

As a side node. I will also change the color picking code, so that it lineary iterates over the frame. That way, the grid size becomes irrelevant, as long as the maximum number of (visible) voxels is below 127*127*127.

Btw. What is your release planning? I'm not involved in the Trove community and I have no idea if anyone or who is using Zoxel.